src/db.py

```diff
@@ -0,0 +1,15 @@
+import sqlite3
+
+
+def create_database():
+    conn = sqlite3.connect("ci.db")
+    cursor = conn.cursor()
+    cursor.execute(
+        "CREATE TABLE history (id INTEGER PRIMARY KEY,commit_hash TEXT, date TEXT, logs TEXT)"
+    )
+    conn.commit()
+    conn.close()
+
+
+if __name__ == "__main__":
```

Comment:
Avoid multiple main functions. There is already a main function in main.py

Reply:
Hmm, i think this is still fine. since this only runs if we run thedb.py file and is used for setting up the database.